web/catrelay.js

```diff
@@ -3,7 +3,7 @@
 const nconf = require('nconf');
 const util = require('util');
 const gcloud = require('gcloud')({
-  projectId: 'cloudcats-next',
+  projectId: process.env.GCLOUD_PROJECT,
```

Comment:
`gcloud` will look for this env var by default, so no need to specify here unless you think being explicit is better for this demo.